### DISCUSSION TITLE
Re-introduce the repositionable modal Window (Dialog Box)

### DISCUSSION THREAD

**Discussion by fabianoriccardi** [microsoft/WindowsAppSDK#4848] [Ideas]
This discussion is to propose the repositionable modal Dialog Window as used to be in previous frameworks ([WPF](https://learn.microsoft.com/en-us/dotnet/desktop/wpf/app-development/dialog-boxes-overview?view=netframeworkdesktop-4.8) and WinForm). The most notable example of what I would implement is the `Options` window in Visual Studio 2022.
I think this kind of window is convenient for windows that requires full user attention, but since they contain a lot of fields, they deserve a full-featured window (movable and resizable). So, the current ContentDialog is not enough for this scenario.

There is already a feature request in [WinAppSDK 1.7 Plans](https://github.com/microsoft/WindowsAppSDK/discussions/4710#discussioncomment-10617112), but in case it won't be implemented in next release, this discussion won't expire.

A related feature that can be useful to implement this is already discussed [here](https://github.com/microsoft/WindowsAppSDK/discussions/3463), but at current state, I didn't find a method to create and populate a window with the aforementioned requirements.

**Comment by castorix** [comment]
You can make ContentDialog movable and resizable
A test with _ManipulationDelta_ to move it and _PointerPressed_, _PointerMoved_, ... on a child rectangle to resize (mouse cursor should be modyfied too... or a better method) (I changed the Template in App.xaml to add a red border) :

![ContentDialogResizable](https://github.com/user-attachments/assets/0a7fbd42-fef8-4ffb-9101-b3954c7f88d0)

**Reply by fabianoriccardi** [reply]
Great, but it doesn't look like a common window...

**Reply by castorix** [reply]
> Great, but it doesn't look like a common window...

Because I updated the Template of ContentDialog (Grid + Border) for the test
